jupyterlab/staging/package.json

```diff
@@ -75,8 +75,8 @@
     "@jupyterlab/markdownviewer": "~4.0.0-alpha.17",
     "@jupyterlab/markdownviewer-extension": "~4.0.0-alpha.17",
     "@jupyterlab/markedparser-extension": "~4.0.0-alpha.17",
-    "@jupyterlab/mathjax2": "~4.0.0-alpha.17",
-    "@jupyterlab/mathjax2-extension": "~4.0.0-alpha.17",
+    "@jupyterlab/mathjax": "~4.0.0-alpha.17",
```

Comment:
Changes in `jupyterlab/staging/package.json` can be reverted because the file is automatically updated during the release process.